handlers/via.go

```diff
@@ -58,3 +87,22 @@ func GetViaConnectedStatus(context echo.Context) error {
 
 	return context.JSON(http.StatusOK, connected)
 }
+
+func GetViaVolume(context echo.Context) error {
+	address := context.Param("address")
+
+	ViaVolume, err := via.GetVolume(address)
+
+	vf := strconv.Itoa(ViaVolume)
+
+	if err != nil {
+		color.Set(color.FgRed)
+		log.Printf("Failed to retreive VIA volume")
+		return context.JSON(http.StatusBadRequest, "Failed to retreive VIA volume")
+	} else {
+		color.Set(color.FgGreen, color.Bold)
+		log.Printf("VIA volume is currently set to %s", vf)
```

Comment:
I would change the '%s' to a '%v' (assuming that you remote the Itoa above). Otherwise you'll get some weird output, something like `%!s(int=1)`